### PR TITLE
Adding aggregated timeseries

### PR DESCRIPTION
**Pull request by spiderjako**
### Added aggregated timeseries metrics functionality
* Generator functionality can be found in aggregatedGenerator.js
* All inner functionality for the aggregated metrics can be found throughout the files:
  - getMetricQueries.js
  - queriesHelper.js
  - getMetricFunctions.js
* All functionality regarding the automatic standart-template generating can be found in:
  - getMetricTestGenerator.js
  - functionsList.js

The rest of the changes are tests, which are automatically generator and also, the automatically generated functions in generatedSheetFunctions.js

### CODE REVIEW

File: generators/aggregatedGenerator.js [hunk 1]
```diff
@@ -0,0 +1,128 @@
+const { metricsList } = require('./getMetricFunctions')
+
+const CURRENCY_DESCRIPTION = `
+* @param {string} currency The metric is calculated using a currency of choice.
+* Available currencies: (currencies)`
+
+const TIMEBOUND_DESCRIPTION = `
+* @param {string} timeBound The metric is calculated only by taking into account the
+* tokens/coins that have moved in the past number of years or days.`
```
Comment (kalkov) [inline generators/aggregatedGenerator.js:9]: `in the past number of years or days` does not sound good to me.

File: generators/aggregatedGenerator.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+const BALANCES_DESCRIPTION = `
+* @param {string} balance Interval of amount of tokens. Example: "0-0.001"`
```
Comment (kalkov) [inline generators/aggregatedGenerator.js:12]: Same here - `Interval of amount of tokens` . Can we have better wording?

File: generators/aggregatedGenerator.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+
+const AGGREGATION_DESCRIPTION = `
+* @param {string} aggregation Aggregation for the timeseries metrics. Example: "LAST"
+`
+
+const FORBIDDEN_METRICS = [ // Aggregated not implemented for these metrics
```
Comment (kalkov) [inline generators/aggregatedGenerator.js:18]: Isn't IGNORED_METRICS a better name?

Comment (spiderjako) [reply generators/aggregatedGenerator.js:18]: Yeah, it would be, I just thought of that name on the go :D ignored is sure better, will change it

**Pull request merged**